Makefile

```diff
@@ -525,8 +526,10 @@ dotestdemos:
 dotest:
 	rm -rf ./tmp/
 	mkdir -p ./tmp/
-    # can't do -n auto due to limits on GPU memory
+  # can't do -n auto due to limits on GPU memory
 	pytest -s --verbose --durations=10 -n 3 --fulltrace --full-trace --junit-xml=build/test-reports/h2o4gpu-test.xml tests_open 2> ./tmp/h2o4gpu-test.$(LOGEXT).log
+	# Test R package
+	/usr/bin/R-3.1.0 -e 'devtools::test("src/interface_r")'
```

Comment:
Is this assuming a specific version of R?

Reply:
Here we only use one version of R to test. Any versions above this should still work (I tried 3.3 and 3.3). The other versions are just not being tested here.

Reply:
Sounds good. Ty